types/messages_encoding.go

```diff
@@ -122,153 +122,3 @@ func (s *SSVMessage) HashTreeRootWith(hh ssz.HashWalker) (err error) {
-	dst = ssz.WriteOffset(dst, offset)
-	offset += len(s.Signature)
-
-	// Offset (2) 'Data'
-	dst = ssz.WriteOffset(dst, offset)
-	offset += len(s.Data)
-
-	// Field (1) 'Signature'
-	if size := len(s.Signature); size > 512 {
-		err = ssz.ErrBytesLengthFn("SignedSSVMessage.Signature", size, 512)
-		return
-	}
-	dst = append(dst, s.Signature...)
-
-	// Field (2) 'Data'
-	if size := len(s.Data); size > 6291893 {
-		err = ssz.ErrBytesLengthFn("SignedSSVMessage.Data", size, 6291893)
-		return
-	}
-	dst = append(dst, s.Data...)
-
-	return
-}
-
-// UnmarshalSSZ ssz unmarshals the SignedSSVMessage object
-func (s *SignedSSVMessage) UnmarshalSSZ(buf []byte) error {
-	var err error
-	size := uint64(len(buf))
-	if size < 16 {
-		return ssz.ErrSize
-	}
-
-	tail := buf
-	var o1, o2 uint64
-
-	// Field (0) 'OperatorID'
-	s.OperatorID = OperatorID(ssz.UnmarshallUint64(buf[0:8]))
-
-	// Offset (1) 'Signature'
-	if o1 = ssz.ReadOffset(buf[8:12]); o1 > size {
-		return ssz.ErrOffset
-	}
-
-	if o1 < 16 {
-		return ssz.ErrInvalidVariableOffset
-	}
-
-	// Offset (2) 'Data'
-	if o2 = ssz.ReadOffset(buf[12:16]); o2 > size || o1 > o2 {
-		return ssz.ErrOffset
-	}
-
-	// Field (1) 'Signature'
-	{
-		buf = tail[o1:o2]
-		if len(buf) > 512 {
-			return ssz.ErrBytesLength
-		}
-		if cap(s.Signature) == 0 {
-			s.Signature = make([]byte, 0, len(buf))
-		}
-		s.Signature = append(s.Signature, buf...)
-	}
-
-	// Field (2) 'Data'
-	{
-		buf = tail[o2:]
-		if len(buf) > 6291893 {
-			return ssz.ErrBytesLength
-		}
-		if cap(s.Data) == 0 {
-			s.Data = make([]byte, 0, len(buf))
-		}
-		s.Data = append(s.Data, buf...)
-	}
-	return err
-}
-
-// SizeSSZ returns the ssz encoded size in bytes for the SignedSSVMessage object
-func (s *SignedSSVMessage) SizeSSZ() (size int) {
-	size = 16
-
-	// Field (1) 'Signature'
-	size += len(s.Signature)
-
-	// Field (2) 'Data'
-	size += len(s.Data)
-
-	return
-}
-
-// HashTreeRoot ssz hashes the SignedSSVMessage object
-func (s *SignedSSVMessage) HashTreeRoot() ([32]byte, error) {
-	return ssz.HashWithDefaultHasher(s)
-}
-
-// HashTreeRootWith ssz hashes the SignedSSVMessage object with a hasher
-func (s *SignedSSVMessage) HashTreeRootWith(hh ssz.HashWalker) (err error) {
-	indx := hh.Index()
-
-	// Field (0) 'OperatorID'
-	hh.PutUint64(uint64(s.OperatorID))
-
-	// Field (1) 'Signature'
-	{
-		elemIndx := hh.Index()
-		byteLen := uint64(len(s.Signature))
-		if byteLen > 512 {
-			err = ssz.ErrIncorrectListSize
-			return
-		}
-		hh.PutBytes(s.Signature)
-		hh.MerkleizeWithMixin(elemIndx, byteLen, (512+31)/32)
-	}
-
-	// Field (2) 'Data'
-	{
-		elemIndx := hh.Index()
-		byteLen := uint64(len(s.Data))
-		if byteLen > 6291893 {
-			err = ssz.ErrIncorrectListSize
-			return
-		}
-		hh.PutBytes(s.Data)
-		hh.MerkleizeWithMixin(elemIndx, byteLen, (6291893+31)/32)
-	}
-
-	hh.Merkleize(indx)
-	return
-}
-
-// GetTree ssz hashes the SignedSSVMessage object
-func (s *SignedSSVMessage) GetTree() (*ssz.Node, error) {
-	return ssz.ProofTree(s)
-}
```

Comment:
won't this be regenrated automatically?
we either need to make an exception for the type or move it to a file that won't cause automatic generation

Reply:
It is not automatically generated because the encoding of the `messages.go` file is not included in the `generate` file. Adding a remark anyway.